stats/Export.md

```diff
@@ -15,7 +15,8 @@ The library SHOULD provide support for multiple types of Aggregations:
 type.
 * `LastValueDataDouble` and `LastValueDataInt64`: data generated for a `LastValue` aggregation based 
 on the `Measure` type.
-* `DistributionData`: data generated for a `Distribution` aggregation.
+* `DistributionData`: data generated for a `Distribution` aggregation. If there is a histogram in 
+the `Distribution`, `DistributionData` MAY contain a list of `Exemplar`s.
```

Comment:
I thought we were moving away from the "specalese" capitalization of random words =)

Reply:
I see, updated.